api/src/main/java/org/arquillian/cube/q/api/NetworkChaos.java

```diff
@@ -0,0 +1,138 @@
+package org.arquillian.cube.q.api;
```

Comment:
But regarding types, I think it is a good place to have all of them, so they are close and you know exactly where to find them.